webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTableHelper.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
+import CheckIcon from '@patternfly/react-icons/dist/js/icons/check-icon';
```

Comment:
These imports won't work at build time, you need to import them from @paternfly/react-icons